carbon-design-system/carbon
# fix(radio-button): top-align wrapping labels

Closes #8380 

#### Changelog

**Changed**

- Updated styling for radio button label
- Updated top margin of radio button element

#### Testing / Reviewing

- Labels should be top-aligned when they wrap
- Ensure the first line is still centered with the round radio button element
- Ensure this doesn't cause any regression in other areas such as the single-select DataTable variant
